packages/layouts-css/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## 0.1.1 (2022-05-25)
+
+### What's new
+
+- **Added `PageLayout` component.** It has sub-components that place content in the correct places.
+- **Added `FluidGrid` component.**
+  - Use it for items (e.g. tiles) array.
+  - It has `--_iui-grid-item-min-width` CSS variable that defines minimum width of the item that is used for responsiveness.
+- **Added `Grid` component.** It is based on CSS grid and has 12 columns. As children pass `iui-layouts-grid-item` sub-component which has:
+  - `--_iui-grid-item-column-span-[BREAKPOINT]` prop that defines how many columns the item should span.
```

Comment:
Are breakpoints specified anywhere in our docs? I think it would be useful for users to know screen size on breakpoint.

Reply:
![image](https://user-images.githubusercontent.com/36186912/171178694-99bc674d-63f1-4203-a041-5223cf4026fa.png)
I don't know where to document CSS part and whether we should do that.

Reply:
Maybe we can add it to CSS README?

Reply:
Anyways, other PR.